File: editor/src/de/fatox/meta/ui/tabs/MetaTab.java
package de.fatox.meta.ui.tabs;

import com.kotcrab.vis.ui.widget.tabbedpane.Tab;

/**
 * Created by Frotty on 16.06.2016.
 */
public abstract class MetaTab extends Tab {

	public MetaTab() {
	}

	public MetaTab(boolean savable) {
		super(savable);
	}

	public MetaTab(boolean savable, boolean closeableByUser) {
		super(savable, closeableByUser);
	}

}

File: editor/src/main/kotlin/de/fatox/meta/ui/tabs/MetaTab.kt
package de.fatox.meta.ui.tabs

import com.kotcrab.vis.ui.widget.tabbedpane.Tab
import de.fatox.meta.Meta.Companion.inject

/**
 * Created by Frotty on 16.06.2016.
 */
abstract class MetaTab : Tab {
    constructor() {
        inject(this)
    }

    constructor(savable: Boolean) : super(savable) {
        inject(this)
    }

    constructor(savable: Boolean, closeableByUser: Boolean) : super(savable, closeableByUser) {
        inject(this)
    }
}